systemd/i3-session.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=i3 Session
+
+Requires=graphical-session-pre.target
+After=graphical-session-pre.target
+BindsTo=graphical-session.target
+Before=graphical-session.target
+
+[Service]
+Type=exec
+ExecStart=i3
```

```suggestion
ExecStart=i3
ExecReload=/usr/bin/i3-msg reload
```
In case the user want's to reload i3.